lib/dialects/abstract/query-generator.js

```diff
@@ -2127,7 +2123,13 @@ var QueryGenerator = {
         comparator = 'IS NOT';
       }
 
-      value = this.escape(value, field);
+      if (_.isPlainObject(value) && value.$any){
+        comparator += ' ANY';
+        value = value.$any;
+        value = '(' + this.escape(value) + ')';
+      } else {
+        value = this.escape(value, field);
+      }
```

Comment:
@mickhansen I didn't put it there because I think that condition only catches ANY not LIKE ANY.  I could write a new condition to catch LIKE/NOTLIKE/ILIKE/NOTILIKE and handle it there, but it seems like it would be a lot of duplicated code from the catch-all condition at the end. Or maybe I missing something else?